code/datums/diseases/anxiety.dm

```diff
@@ -36,6 +36,14 @@
 			if(prob(2))
 				affected_mob.visible_message("<span class='danger'>[affected_mob] coughs up butterflies!</span>", \
 													"<span class='userdanger'>You cough up butterflies!</span>")
-				new /mob/living/simple_animal/butterfly(affected_mob.loc)
-				new /mob/living/simple_animal/butterfly(affected_mob.loc)
-	return
+				for(var/i in 1 to 2)
+					var/mob/living/simple_animal/butterfly/B = new(affected_mob.loc)
+					addtimer(CALLBACK(B, /mob/living/simple_animal/butterfly/.proc/decompose), rand(5, 25) SECONDS)
+
+/**
+ * Made so severe anxiety does not overload the SSmob while keeping it's effect
+ */
+/mob/living/simple_animal/butterfly/proc/decompose()
+	visible_message("<span class='notice'>[src] decomposes due to being outside of its original habitat for too long!</span>", \
+					"<span class='userdanger'>You decompose for being too long out of your habitat!</span>")
```

Comment:
May as well point this out. LGTM otherwise.
```suggestion
	visible_message("<span class='notice'>[src] decomposes due to being outside of its original habitat for too long!</span>",
					"<span class='userdanger'>You decompose for being too long out of your habitat!</span>")
```

Reply:
Oh yeah I keep forgetting you can do this